webhook/resourcesemantics/validation/validation_admit_test.go

```diff
@@ -645,7 +645,7 @@ func TestNewResourceAdmissionController(t *testing.T) {
 
 	NewAdmissionController(
 		ctx, testResourceValidationName, testResourceValidationPath,
-		handlers,
+		nil,
```

Comment:
I'm curious why you need these changes?

Reply:
leftover from earlier iterations, thanks!